lib/front_end_builds/ext/routes.rb

```diff
@@ -6,7 +6,6 @@ class Mapper
     # Create a front end in your rails router.
     def front_end(name, path = name, options = {})
       defaults = {
-          branch: 'master',
```

Comment:
This is great. I think we're really close to the point where we can start to build routes dynamically so there isn't the need for router editing.